src/main/resources/application.properties

```diff
@@ -14,6 +14,7 @@ chBankAccName=${CH_BANK_ACC_NAME}
 penalty.allowed-ref-starts-with[0]=A
 penalty.allowed-ref-starts-with[1]=PN
 penalty.ref-starts-with-path=/late-filing-penalty/ref-starts-with
+penalty.ref-starts-with-url=${chs.url}${penalty.ref-starts-with-path}
```

Comment:
Could we use the ${CHS_URL} here instead?

Reply:
Yep, we could use both because it's the first property we set on line 1.
`chs.url=${CHS_URL}`

I prefer to keep the way I have it as we use that property in other places like the footer anyway.